lib/routes/ncm/djradio.js

```diff
@@ -3,22 +3,68 @@ const got = require('@/utils/got');
 module.exports = async (ctx) => {
     const id = ctx.params.id;
 
-    const response = await got({
-        method: 'post',
-        url: 'http://music.163.com/api/dj/program/byradio',
-        headers: {
-            Referer: 'https://music.163.com/',
-        },
-        form: {
-            radioId: id,
-            limit: 1000,
-            offset: 0,
-        },
-    });
+    const ProcessFeed = async (limit, offset) =>
+        await got({
+            method: 'post',
+            url: 'http://music.163.com/api/dj/program/byradio',
+            headers: {
+                Referer: 'https://music.163.com/',
+            },
+            form: {
+                radioId: id,
+                limit,
+                offset,
+            },
+        });
 
-    const programs = response.data.programs || [];
+    const response = await ProcessFeed(1, 0);
 
+    const programs = response.data.programs || [];
     const { radio, dj } = programs[0] || { radio: {}, dj: {} };
+    const count = response.data.count || 0;
+
+    const countPage = [];
+    for (let i = 0; i < Math.ceil(count / 500); i++) {
+        countPage.push(i);
+    }
+
+    const items = await Promise.all(
+        countPage.map(async (item) => {
+            const response = await ProcessFeed(500, item * 500);
+            const programs = response.data.programs || [];
+            const list = programs.map((pg) => {
+                const image = `<img src=${pg.coverUrl} />`;
+                const description =
+                    '<div>' +
+                    (pg.description || '')
+                        .split('\n')
+                        .map((p) => `<p>${p}</p>`)
+                        .join('') +
+                    '</div>';
+                const src = 'https://music.163.com/program/' + pg.id;
+                const duration = ~~(pg.duration / 1000);
+                const itunes_duration = `${(duration / 60).toFixed(0).padStart(2, '0')}:${(duration % 60).toFixed(0).padStart(2, '0')}`;
+                const html =
+                    image + description + `<div><audio src="http://music.163.com/song/media/outer/url?id=${pg.mainTrackId}.mp3" controls="controls"></audio><p>时长: ${itunes_duration}</p><p><a href="${src}">查看节目</a></p></div>`;
+
+                return {
+                    title: pg.name,
+                    link: src,
+                    pubDate: new Date(pg.createTime).toUTCString(),
+                    published: new Date(pg.createTime).toISOString(),
```

Comment:
```suggestion
                    pubDate: parseDate(pg.createTime),
                    published: parseDate(pg.createTime),
```
Please use parseDate()
ref: #6673
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei